common-utils/src/common/deployment/cli.py

```diff
@@ -18,36 +19,53 @@
 
 
 def parse_args(args: list[str]) -> Namespace:
-    """Basic CLI setup for running deploymebt utils as needed.
+    """Basic CLI setup for running deployment utils as needed.
 
     Args:
-        args (_type_): _description_
+        args (list[str]): List of args passed from cli execution.
 
     Returns:
-        _type_: _description_
+       Namespace: parsed args object
     """
-    """Basic CLI for running deploymebt utils as needed."""
     parser = ArgumentParser(
-        description="""Running this cli directly will:
-    - Build and push your Docker images to our private ECR repository.
-    - Create an S3 filesytem block in Prefect v2 for storing flows and
-    for storing flow run staging/working files.
+        description="""basic CLI setup for running deployment utils as needed.
     """,
         formatter_class=RawTextHelpFormatter,
     )
-    parser.add_argument(
+    subparsers = parser.add_subparsers(
```

Comment:
I hadnt noticed the CLI before. This is nice and I am sure it will help the project be generally more approachable for more devs.